src/vos/tests/evt_ctl.c

```diff
@@ -78,6 +78,14 @@ static daos_handle_t		ts_toh;
 #define EVT_SEP_EXT		'-'
 #define EVT_SEP_EPC		'@'
 
+/* Data sizes */
+#define D_1K_SIZE		1024
```

Comment:
Should as add lower size as well like bytes? Will single byte will work ?

Reply:
There are several other tests which does non-standard bytes. single byte is also covered.